_inc/connect-button.js

```diff
@@ -42,11 +42,17 @@ jQuery( document ).ready( function( $ ) {
 			}
 		},
 		startConnectionFlow: function() {
-			var abTestName = 'jetpack_connect_in_place_v2';
+			var abTestName = 'jetpack_connect_in_place_v3';
+			var type = jpConnect.identity && jpConnect.identity._ut;
+			var identity = jpConnect.identity && jpConnect.identity._ui;
 			$.ajax( {
 				url: 'https://public-api.wordpress.com/wpcom/v2/abtest/' + abTestName,
 				type: 'GET',
 				error: jetpackConnectButton.handleConnectionError,
+				data: {
```

Comment:
Instead of parsing these and assigning to `type` and `identity` vars, does it make sense to pass just `jpConnect.identity` here?

Reply:
Yes great idea!